Assignment/oddeven.txt

```diff
@@ -0,0 +1,21 @@
+/* [16] WAP to check number is Even or Odd
+ID : 23BCA261 */
+
+#include<iostream.h>

```

Comment:
why you add iostram.h file?

Reply:
iostream.h header file is included to use cout and cin object. it is created in iostram.h header file. 

this header file contain "iostream" class and create object of this class "cout" and "cin"